lib/main.dart

```diff
@@ -74,6 +75,8 @@ class TMailApp extends StatelessWidget {
       unknownRoute: AppPages.unknownRoutePage,
       defaultTransition: Transition.noTransition,
       initialRoute: AppRoutes.home,
-      getPages: AppPages.pages);
+      getPages: AppPages.pages,
+      builder: FlutterSmartDialog.init(),
```

Comment:
why we need use it here? maybe we use it somewhere before?

Reply:
To register the 3rd party library `loading bar`. We have implemented it on the `master` branch